### PR TITLE
Install the production Manage instance

### PR DESCRIPTION
**Pull request by quartje**
This PR will install a second installation of Manage in the SP Dashboard VM. This instance is called manage-prod (sometimes referred to in the ansible tasks as manage-prd). The instance will be used to publish production entities to from SP Dashboard.

### CODE REVIEW

File: ansible/group_vars/all.yml [hunk 1]
```diff
@@ -120,12 +120,10 @@ maven_snapshot_repo: https://build.openconext.org/repository/public/snapshots
 error_subject_prefix: "[{{ ansible_hostname }}] "
 
 # Some engineblock variables are shared between applications
-engine_api_domain: engine-api.{{ base_domain }}
```
Comment (MKodde) [inline ansible/group_vars/all.yml]: Thanks for cleaning up after me :+1:

**Comment by quartje** [comment]
I've fixed it by correctingthe handler name. It should work now

**Comment by MKodde** [comment]
Just tested your changes and all tasks where performed successfully.

Some notes to get everything up and running, and please append instructions where needed:

1. Update your box with `vagrant provision`. :coffee: 
2. Log in on mananage and import the new manage-prod sp. Do this by:
    1. Export the manage metadata
    2. Go to the manage sp
    3. Go to the export tab and copy the metadata xml
    4. Go to the import page via the main menu Import button
    5.  Import the metadata you have on your clipboard and update all `https://manage.dev.support.surfconext.nl` with `https://manage-prod.dev.support.surfconext.nl`
    6. Update some of the metadata untill you are allowed to save the new manage instance. I removed some of the contact person entries and set a custom name to identify the entity.
    7. Push the metadata changes to EngineBlock
3. You should now be able to log in to manage-prod

:tada:  

@pablothedude can you verify this?

**Comment by pablothedude** [comment]
Thanks for this walktrough this helped me a lot and worked like a charm. The removal of the technical contacts however wasn't necessary and the  prod hostname should be added to /etc/host. I will add this request to https://github.com/SURFnet/sp-dashboard/pull/163

**Pull request merged**